src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryExcelTemplate.java

```diff
@@ -207,17 +209,74 @@ private void createDetailSheet(final IndicatorDTO indicator) {
 		sheetEx.addMergedRegion(new CellRangeAddress(rowIndex, rowIndex, 1, numbOfCols));
 
 		// data entry
+        //for year above months in indicators
```

Comment:
@manishvishnoi2 : This needs an indentation fix before the pull request can be merged.